pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -435,28 +436,31 @@ func doActivate(ctx *zedmanagerContext, uuidStr string,
 		if remaining < need {
 			var errStr string
 			var entities []*types.ErrorEntity
-			errSeverity := types.ErrorSeverityWarning
-			retryCondition := "Retry will be triggered when apps will shutdown: "
+			var domains []string
+			errSeverity := types.ErrorSeverityError
 			if remaining+halting < need {
```

Comment:
In this case we still want to list a retry condition which says it will retry once or more of the running app instances halt. That list should exclude ourselves, and exclude app instances which are !Activated.

Reply:
I move collecting of domains few lines below, so they will be accounted